package.json

```diff
@@ -77,6 +77,7 @@
     "lodash.upperfirst": "4.3.1",
     "material-design-icons": "3.0.1",
     "mustache": "2.3.0",
+    "netlify-cli": "1.2.2",
```

Comment:
update: keeping this for now until the site gets to MVP; need to retain ability to push components out to the world.